charts/backingservices/README.md

```diff
@@ -1,85 +1,9 @@
 # BackingServices Helm chart
 
-The Pega Infinity backing service is a feature which you can deploy as an independent service module. For example `Search and Reporting Service` or `SRS` backing service can replace the embedded search feature of Pega Infinity Platform. To use it in your deployment, you provision and deploy it independently as an external service which provides search and reporting capabilities with a Pega Infinity environment.  
+The Pega Infinity backing service Helm charts are used to deploy dedicated services to support Pega Infinity as an independent microservice. To use these services in your deployment, deploy the backing services chart independently following the instructions below.  
```

Comment:
minor suggestion to get rid of passive voice>>
You deploy dedicated services to support Pega Infinity as an independent microservice using the Pega Infinity backing service Helm charts. To use these services, deploy the backing services chart independently following the instructions below.

Reply:
Updated